src/app/pages/storage/storage.page.ts

```diff
@@ -17,6 +18,65 @@ import { forkJoinWithDefault } from 'src/app/utils/rx-operators';
 export class StoragePage {
 
   private readonly proofs$ = this.proofRepository.getAll$();
+
+
+  readonly today$ = new Observable(observer => {
```

Comment:
This observable emits `''` after 1000 ms. I don't think it's relevant to "today" and it looks unnecessary here.